e2e/scenario_test.go

```diff
@@ -519,7 +519,7 @@ func Test_ubuntu1804(t *testing.T) {
 			Cluster: ClusterKubenet,
 			VHD:     config.VHDUbuntu1804Gen2Containerd,
 			LiveVMValidators: []*LiveVMValidator{
-				containerdVersionValidator("1.7.1+azure-1"),
+				containerdVersionValidator(getExpectedPackageVersions("containerd", "ubuntu", "r1804")[0]),
 				runcVersionValidator("1.1.14-1"),
```

Comment:
Could you do the same for runc ?

Reply:
Added, but there are differences between what `--version` command returns for containerd/runc and what we store in `components.json`. Will sync with Devin on Monday to discuss